### ISSUE TITLE
Cygwin does not create its core files

### ISSUE THREAD

**Issue by 0zhu** [vegardit/cygwin-portable-installer#1]
Hello Vegardit,

First of all I would like to thank you for such a great tool, always wanted to have portable cygwin+comemu stack.

I wanted to ask if you experienced any similar issues while installing Cygwin portable.
I recently tried to install it on my PC (Windows 10 x64) and found that Cygwin does not create its core files during setup process. It hangs on "Replacing etc/fstab" stage and goes to :fail afterwards.
After this I looked into %CYGWIN_ROOT% folder and found that there are no core files (/home, /bin, etc.)
Only:
./etc/setup/setup.rc
./etc/setup/timestamp
./setup-x86_64.exe

Tried different mirrors, creating fstab file manually, running installation process from virtualbox VM with no success. The weird thing is that one week ago cygwin+conemu was installed without issues on my friend's PC (also Windows 10 x64) 

Any help would be highly appreciated.

---------------------
Regards,
Alex

**Comment by sebthom** [comment]
Hi Alex,
I just tried it again on Windows 7 and 10 Pro 64bit. The installation works flawlessly. 

I used to have issues with 3rd party Anti Virus scanners, so in case you have one installed you may want to deactivate it during the installation to see if that solves your issue.

You can also remove the `@echo off` statement from the top of the batch file so you see exactly which command fails.

Regards,
Seb

**Comment by 0zhu** [comment]
Hello Sebastian,

During further investigation it was found that the problem was caused by existing non-portable installation of ConEmu and Cygwin (first I thought it was due to new Creators Update).
For some reason it didn't allow Cygwin installer to install its core files properly and process hung on *"replacing /etc/fstab"* stage (however no suspicious entries was found in verbose Cygwin installer output).

I [deleted Cygwin](https://cygwin.com/faq/faq.html#faq.setup.uninstall-all) and ConEmu (Cygwin didn't have `cygrunsrv`, so the manual deletion of everything related to Cygwin and ConEmu in registry and Windows search was enough).
After it had been completed, I rebooted the PC and everything was installed properly via `cygwin-portable-installer.cmd`

Thanks again for your support!

----------------------------
Regards,
Alex
